packages/web/examples/express/package.json

```diff
@@ -42,9 +42,6 @@
     "prelint": "prettier --trailing-comma all --write './**/*.md'",
     "lint": "eslint --ext ts --ignore-pattern dist .",
     "prebuild": "yarn clean",
-    "prepare": "yarn build",
-    "prepublishOnly": "yarn lint",
-    "preversion": "yarn lint",
```

Comment:
You just removed them here?

Reply:
yes, it's an example repo, not something that would ever published